source/envs/env_avoid_observber.py

```
@@ -0,0 +1,609 @@
+        self.camera_size_tiles = (20, 10)
+        self.camera_size_px = (
+            self.camera_size_tiles[0] * tile_size,
+            self.camera_size_tiles[1] * tile_size,
+        )
+
+        # Speeds
+        self.agent_speed = 20  # agent is twice observer speed
+        self.observer_speed = 10
+        self.observer_pause_frames = 30  # 1초 대기 (30fps 기준)
+
+        obs_dim = 2 + 2 * num_observers
```

Ruff (0.12.2)

Local variable `obs_dim` is assigned to but never used

Remove assignment to unused variable `obs_dim`

(F841)